lib/charms/mongodb/v1/helpers.py

```diff
@@ -143,6 +143,9 @@ def get_mongod_args(
         # for simplicity we run the mongod daemon on shards, configsvrs, and replicas on the same
         # port
         f"--port={Config.MONGODB_PORT}",
+        "--auditDestination file",
```

Comment:
lets be consistent with delimiter for all options
```suggestion
        "--auditDestination=file",
```

Reply:
Done, thank you